### PR TITLE
Delay opening of the popup to correctly align with window insets. Fixes #400

### LINKED ISSUE #400: remove popup, top elevation
hi  
how i can remove the top Elevation of popup window ?
image

edit : 
its not elevation of popupwindow its elevation of keyboard ? why keyboard must be open when 
emojPopupWindow is open ?

### PR DESCRIPTION
**Pull request by mario**
250ms is the amount of time the system needs to apply window insets that we set. Therefore, we need to delay opening of the popup by 250ms the first time we do it.

Signed-off-by: Mario Danic <mario@lovelyhq.com>

### CODE REVIEW

**Comment by mario** [comment]
I don't even know how to fix the "magic number" lint so please do it :P Thanks.

**Comment by mario** [comment]
Can we merge?

File: emoji/src/main/java/com/vanniktech/emoji/EmojiPopup.java [hunk 1]
```diff
@@ -331,8 +342,12 @@ public void dismiss() {
...
   void showAtBottom() {
     isPendingOpen = false;
-    popupWindow.showAtLocation(rootView, Gravity.NO_GRAVITY, 0,
-        Utils.getProperHeight(context) + popupWindowHeight);
+    editText.postDelayed(new Runnable() {
+      @Override public void run() {
+        popupWindow.showAtLocation(rootView, Gravity.NO_GRAVITY, 0,
+            Utils.getProperHeight(context) + popupWindowHeight);
+      }
+    }, delay);
```
Comment (rubengees) [inline emoji/src/main/java/com/vanniktech/emoji/EmojiPopup.java]: Is this safe in regard to race conditions?

Comment (mario) [reply emoji/src/main/java/com/vanniktech/emoji/EmojiPopup.java]: No. Got better options? :)

Comment (rubengees) [reply emoji/src/main/java/com/vanniktech/emoji/EmojiPopup.java]: Sorry, no, I can't really follow the flow in `EmojiPopup` anymore :/

**Comment by mario** [comment]
The issue is I'm not sure I want to merge this :(

**Comment by mario** [comment]
I will look into this tonight.

**Comment by mario** [comment]
@vanniktech let's get this in, I don't have a better solution for now.

**Pull request merged**